app/scenes/GroupMembers/GroupMembers.tsx

```diff
@@ -59,7 +59,8 @@ function GroupMembers({ group }: Props) {
         <>
           <Text type="secondary">
             <Trans
-              defaults="Add and remove team members in the <em>{{groupName}}</em> group. Adding people to the group will give them access to any collections this group has been added to."
+              // TODO this string can use improvement
+              defaults="Add and remove members to the <em>{{groupName}}</em> group. Adding people to the group will give them access to any collections this group has been added to."
```

Comment:
```suggestion
              defaults="Add and remove members to the <em>{{groupName}}</em> group. Members of the group will have access to any collections this group has been added to."
```

Slightly better. This makes me think, really need the reverse view of permissions so you can see which collections a group has been added to.